frontend/geeta-chopra-award-explorer/style.css

```
@@ -0,0 +1,635 @@
+    text-align: center;
+    color: #fff;
+    padding: 0 2rem;
+}
+
+.lightbox-caption h3 {
+    margin: 0 0 0.25rem;
+    font-size: 1.1rem;
+}
+
+.lightbox-caption p {
+    margin: 0;
```

Stylelint (17.14.0)

[error] Duplicate property "margin" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)